styleguide.config.js

```diff
@@ -145,6 +145,11 @@ module.exports = {
             name: "Popover",
             content: "packages/wonder-blocks-popover/docs.md",
             sections: [
+                {
+                    components: [
+                        "packages/wonder-blocks-popover/components/popover.js",
+                    ],
+                },
                 {
                     name: "Building Blocks",
```

Comment:
This section should have `private: true`

Reply:
Not necessarily, as this section is part of the public API. Maybe we could include `Internal Components` (PopoverDialog, CloseButton, PopoverContext, etc) and make it private. The only thing I see is that I find it a bit unnecessary to expose this information inside React Styleguidist. Personally, I think that consumers wouldn't need to know the details of the private API.

Are we using these private sections only for development purposes (localhost), right? Asking this mostly because I'm not sure how needed is to include this information into Styleguidist. What are your thoughts?

Reply:
As per our last conversation, I changed the structure here to keep it consistent with the rest of the packages. Thanks for the context!